### PR TITLE
[RHACS][Docs][4.6 and later versions] ROX-26679: Compliance reporting documentation

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.6+

Issue:
https://issues.redhat.com/browse/ROX-26679

Link to docs preview:
https://84560--ocpdocs-pr.netlify.app/openshift-acs/latest/operating/manage-compliance/scheduling-compliance-scans-and-assessing-profile-compliance.html

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-main` and cherry-pick to:
    - `rhacs-docs-4.6`

### CODE REVIEW

File: modules/schedules-page-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
++
+The report has been successfully emailed.
+
+** `Error`
++
+There was an issue with the report job.
+
+** `None`
++
+There are no recent jobs available.
```
Comment (xenolinux) [inline modules/schedules-page-overview.adoc:42]: This list might be represented in the form of table(s).

**Review by xenolinux — COMMENTED** [review]
One tiny nit ; two (optional) comments - Otherwise LGTM! Nice work.

File: modules/analyzing-compliance-scan-schedules.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
++
+You can configure the following setting for scan schedule reports:
++
+*** *Vulnerability report job history retention*: The number of days to retain your scan schedule report.
+.. To edit this value, click *Edit*, make your changes, and then click *Save*.
++
+[NOTE]
+====
+All report jobs are kept in your system until they exceed the limit you set.
+====
```
Comment (lvalerom) [inline modules/analyzing-compliance-scan-schedules.adoc:45]: This is not implemented in 4.6. I think we should remove this section.

**Review by sachaudh — APPROVED** [review]
I agree with Luis' comments. LGTM otherwise

**Review by lvalerom — APPROVED** [review]
LGTM

**Pull request merged**